### ISSUE TITLE
Output video longer than preview

### ISSUE THREAD

**Issue by Wittner** [AliaksandrSiarohin/first-order-model#386]
I am creating a deepfake video using your code through Google Colab. Everything works fine until the download step. I am really happy with the results I get and everything looks normal on the preview, the original png shows up, beside it the 'driver' video and the final output on the right hand side, all in sync and all at proper speed. The video I am working on lasts 21 seconds, but when I download it, the resultant video is 34 seconds long. Any idea why it doubles in length?

**Comment by AliaksandrSiarohin** [comment]
Yes, it saves it with fixed fps. Check the fps of your video, and save it with fps that you want. Et. ```imageio.mimsave(..., fps=30)```

**Comment by WatchfulFox** [comment]
Thank you so much. Will do that.

**Comment by Wittner** [comment]
Thank you so much. Will do that. Sorry, commented under the wrong account :)

**Comment by RobertSmith8** [comment]
Hi I have a similarly problem the "driver" has fps = 30 but the preview and DL video has only fps = 20. My 35 sec video results in an 52 sec video.

I tried to change
```
fps = reader.get_meta_data()['fps']
to
fps = 30

Also tried with
imageio.mimsave('../generated.mp4', [img_as_ubyte(frame) for frame in predictions], fps=30)
```
But with no success how can I produce a video with same FPS as original (30)?

The Source/Driver is 
https://www.youtube.com/watch?v=WM0tMBmiKDc&ab_channel=RyomaHoshi%E2%80%A2

DL using YTD Video Download


**Problem solved**
I find generated.mp4 with correct fps and length :) 


<img width="319" alt="Screenshot 2021-03-31 at 18 16 31" src="https://user-images.githubusercontent.com/3736516/113177096-751e7700-924d-11eb-9193-8d09674b88ef.png">
